packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -74,15 +60,50 @@ class LayoutProperties {
 
   LayoutProperties get largestHeightChild => _largestHeightChild;
 
-  double get smallestWidthChildPercentage => _smallestWidthChildPercentage;
+  double get smallestWidthChildPercentage => _smallestWidthChild.width / width;
```

Comment:
nit: while you are here change all uses of Percentage to Fraction
as percentage means a number from 0-100 while these are all from 0-1